packages/processors/src/processors/registry/handlers/profileMetadataUpdated.handler.ts

```diff
@@ -0,0 +1,65 @@
+import { Changeset, ProjectType } from "@grants-stack-indexer/repository";
+import { ChainId, ProcessorEvent } from "@grants-stack-indexer/shared";
+
+import { IEventHandler, ProcessorDependencies } from "../../../internal.js";
+import { ProjectMetadata, ProjectMetadataSchema } from "../../../schemas/index.js";
+
+type Dependencies = Pick<
+    ProcessorDependencies,
+    "projectRepository" | "evmProvider" | "metadataProvider" | "logger"
+>;
+/**
+ * Handles the ProfileMetadataUpdated event for the Registry contract from Allo protocol.
+ */
```

Comment:
i like adding here a general overview of the main actions performed by the handler

for example for `PoolCreatedHandler`
```
/**
 * Handles the PoolCreated event for the Allo protocol.
 *
 * This handler performs the following core actions when a new pool is created:
 * - Retrieves the metadata associated with the pool
 * - Determines the correct token address, handling native tokens appropriately.
 * - Extracts the correct strategy information from the provided strategy ID.
 * - Calculates the funded amount in USD based on the token's pricing.
 * - Creates a new round object
 */
  ```

Reply:
this is great, i saw it on your prs 💯